src/language/tekster_en.ts

```diff
@@ -0,0 +1,105 @@
+import { Tekster } from './typer';
+
+export const en: Tekster = {
+  'side.tittel': 'Your benefits for single parents',
+  'side.undertittel': 'Transitional benefit, childcare benefit, and support for school fees',
+
+  'stonader.tittel': 'Your benefits',
+  'stonader.ingenFunnet':
+    'We found no payment periods related to benefits for single mother or father.',
+
+  'endreEllerEttersende.tittel': 'Do you need to report changes or submit documentation?',
+
+  'endre.tittel': 'Report a change',
+  'endre.brødtekst': 'Here you can report changes that may affect your benefit.',
+
+  'ettersende.tittel': 'Submit documentation',
+  'ettersende.brødtekst': 'Here you can submit documents for your case.',
+
+  'dokumenter.tittel': 'Your documents',
+  'dokumenter.ingenFunnet':
+    'We found no documents to show related to benefits for single mother or father. Note that only documents and messages you have submitted digitally will be displayed here.',
+  'dokumenter.seAlle': 'See all documents',
+  'dokumenter.henter': 'Fetching documents',
+  'dokumenter.galt': 'Something went wrong while fetching your documents.',
+
+  'søke.tittel': 'Do you want to apply?',
+
+  'overgangsstønad.tittel': 'Transitional benefit',
+  'overgangsstønad.brødtekst':
+    'Ensures you income for up to 3 years when you have at least 60 per cent of the daily care for children under 8 years of age.',
```

Comment:
rettelse til "your", "percent", og "if" i stedet for "when" (mere betinget): "Ensures your income for up to 3 years if you have at least 60 percent of the daily care for children under 8 years of age".

Reply:
Beholder 'Ensures', men endrer resten